dwave_micro_client_dimod/tiling.py

```diff
@@ -0,0 +1,167 @@
+"""
+TilingComposite
+==================
+"""
+from __future__ import division
+from math import sqrt, ceil
+from dwave_micro_client_dimod.sampler import Structure
+
+import dimod
+import dwave_networkx as dnx
+import dwave_embedding_utilities as embutil
+
+__all__ = ['TilingComposite']
+
+class TilingComposite(dimod.TemplateComposite):
+    """Composite to tile a small problem across a structured sampler.
+
+    Args:
```

Comment:
It would be good to describe here in general what Tiling is. I think a sentence or two would suffice.

Reply:
Also that this only makes sense for Chimera-structured samplers

Reply:
Good call. Done.